cclient/api/core/get_transactions_to_approve.h

```diff
@@ -17,20 +26,19 @@ extern "C" {
 #endif
 
 /**
- * Tip selection which returns trunkTransaction and branchTransaction. The input
- * value is depth, which basically determines how many bundles to go back to for
- * finding the transactions to approve. The higher your depth value, the more
- * "babysitting" you do for the network (as you have to confirm more
- * transactions).
- *
- * https://iota.readme.io/reference#gettransactionstoapprove
+ * @brief Does the <b>tip selection</b>
  *
- * @param service IRI node end point.
- * @param req Request containing number of bundles to go back to determine
- * the transactions for approval.
- * @param res Response containing transactions to approve (branch + trunk)
+ * Tip selection which returns <b>trunk</b> and <b>branch</b> transaction. The input value <b>depth</b>
+ * determines how many milestones to go back for finding the transactions to approve. The higher your <b>depth</b>
+ * value, the more work you have to do as you are confirming more transactions. If the <b>depth</b> is too large
+ * (usually above 15, it depends on the node's configuration) an error will be returned. The <b>reference</b> is an
+ * optional hash of a transaction you want to approve. If it can't be found at the specified <b>depth</b> then an
```

Comment:
you want to approve --> you want the selected tips to reference

Reply:
To approve is referenced by two tips in short. it's interchangeable in the white paper.